src/Cli/dotnet/commands/dotnet-workload/install/WorkloadManifestUpdater.cs

```diff
@@ -42,7 +43,9 @@ internal class WorkloadManifestUpdater : IWorkloadManifestUpdater
             string tempDirPath,
             IWorkloadInstallationRecordRepository workloadRecordRepo,
             PackageSourceLocation packageSourceLocation = null,
-            Func<string, string> getEnvironmentVariable = null)
+            Func<string, string> getEnvironmentVariable = null,
+            bool displayManifestUpdates = true
+            )
```

Comment:
Small nit: I would put the parenthesis on the same line as the last parameter.
```suggestion
            bool displayManifestUpdates = true)
```